### ISSUE TITLE
Multiple unpatched critical/high CVEs

### ISSUE THREAD

**Issue by k3an3** [gogs/gogs#7777]
As per https://www.sonarsource.com/blog/securing-developer-tools-unpatched-code-vulnerabilities-in-gogs-1/, multiple critical and high CVEs were reported to Gogs maintainers in April 2023 but have not been addressed:

- Argument Injection in the built-in SSH server (CVE-2024-39930, CVSS 9.9 Critical)
- Argument Injection when tagging new releases (CVE-2024-39933, CVSS 7.7 High)
- Argument Injection during changes preview (CVE-2024-39932, CVSS 9.9 Critical)
- Deletion of internal files (CVE-2024-39931, CVSS 9.9 Critical)
- Remote code execution via web editor when editing and moving a symlink (CVE-2024-44625, I'm guessing CVSS ~8.8 High)

Is this project still alive? Strongly considering switching to Gitea where these issues are not present.

**Comment by Elikill58** [comment]
> Is this project still alive?

I think no

**Comment by fnetX** [comment]
I am a member of the [Forgejo](https://forgejo.org) security team, and we have tried to inform Gogs about vulnerabilities we discovered in our codebase and traced back to Gogs. However, we haven't had much luck either. Shortly after we reported a critical vulnerability, the [security policy of Gogs was updated](https://github.com/gogs/gogs/commit/61940ca879d2599e446d302f0134dac2d08ce2fe) and does no longer accept reports via email, so we'll likely stop bothering about Gogs.
